osu.Game/Overlays/Mods/ModSection.cs

```diff
@@ -127,18 +127,28 @@ public void DeselectTypes(IEnumerable<Type> modTypes, bool immediate = false)
         }
 
         /// <summary>
-        /// Select one or more mods in this section and deselects all other ones.
+        /// Updates all buttons with the given list of selected mods.
         /// </summary>
-        /// <param name="modTypes">The types of <see cref="Mod"/>s which should be selected.</param>
-        public void SelectTypes(IEnumerable<Type> modTypes)
+        /// <param name="newSelectedMods">The types of <see cref="Mod"/>s to select.</param>
```

Comment:
No longer just the types.

Reply:
sounds good?
```suggestion
        /// <param name="newSelectedMods">The new list of selected mods to set from.</param>
```

Reply:
"set from" sounds weird, would just keep it simple and keep "select".